Specs/Core/getStringFromTypedArraySpec.js

```diff
@@ -14,17 +14,23 @@ defineSuite([
         expect(string).toEqual('');
     }
 
+    var previous;
+    beforeEach(function() {
+        previous = getStringFromTypedArray.decode;
+    });
+
+    afterEach(function() {
+        getStringFromTypedArray.decode = previous;
+    });
+
     it('converts a typed array to string', function() {
         verifyString();
     });
 
     it('converts a typed array to string when forced to use fromCharCode', function() {
-        var previous = getStringFromTypedArray.decode;
         getStringFromTypedArray.decode = getStringFromTypedArray.decodeWithFromCharCode;
```

Comment:
This should use a spy instead of the manual beforeEach/afterEach above.  Just spy on `decode` and have it call `getStringFromTypedArray.decodeWithFromCharCode`.